src/index.ts

```diff
@@ -235,6 +235,14 @@ https://blog.newrelic.com/product-news/aws-lambda-extensions-integrations/
     }
 
     const { exclude = [], include = [] } = this.config;
+
+    if (!this.config.apiKey || this.config.apiKey === "") {
```

Comment:
Isn't this redundant? Doesn't ! handle the empty string case?

Reply:
Updated--also, this change is scoped to just this API key handling now.